README.md

```diff
@@ -47,9 +48,17 @@ To run this sample you will need the following:
 
 1. In the list of pages for the app, select **Authentication**.
     - In the **Redirect URIs** section, select **Web** in the combo-box and enter the following redirect URI:
-    `http://localhost:3000/auth/openid/return`
-    - In the **Advanced settings** section, set **Logout URL** to `http://localhost:3000`.
-    - In the **Advanced settings > Implicit grant** section, check **ID tokens** as this sample requires the [Implicit grant flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) to be enabled to sign-in the user.
+    `http://localhost:3000/auth/openid/return` (:warning:)
+    - In the **Advanced settings > Implicit grant and hybrid flows** section, check **ID tokens** as this sample requires the [hybrid flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) to be enabled to sign-in the user.
+    - Select **Save**.
+
+> :warning: In this tutorial sample, we use HTTP schemes as the application runs on locahost only. However, HTTP schemes are insecure and should not be used in production.
```

Comment:
What are HTTP schemes. do you want to provide a link?
Do you mean HTTP whereas HTTPS would be recommended?

Reply:
Link to best practices added.